base/doc/ltnews38.tex

```diff
@@ -180,44 +310,50 @@ \section{Extended cross-referencing of document properties}
 \end{verbatim}
 To reference the value the \cs{RefProperty} command is provided which takes 
 two arguments, the label and the property: 
 \begin{verbatim}
 \RefProperty{mylabel}{chapter}
 \end{verbatim}
  
-The new module predeclares a set of generally useful properties: 
-\texttt{currentlabel} (the standard counter representation), \texttt{page} 
-(the page representation), \texttt{title} (the title if set e.g.~by 
-\pkg{nameref}), \texttt{target} (the name of the link target if set e.g.~by 
-\pkg{hyperref}), \texttt{pagenum} (the page as an arabic number), 
-\texttt{abspage} (the absolute page number), \texttt{counter} (the name of 
-the current counter), \texttt{xpos} and \texttt{ypos} the position on the 
-page as set by a previous \cs{pdfsavepos}. 
+The new module predeclares a set of generally useful properties:
+\texttt{label} (the standard counter representation), \texttt{page}
+(the page representation), \texttt{title} (the title if set e.g.~by
+\pkg{nameref}), \texttt{target} (the name of the link target if set,
+e.g., by \pkg{hyperref}), \texttt{pagetarget} (the name of target
+added each page by \pkg{hyperref}), \texttt{pagenum} (the page as an
+arabic number), \texttt{abspage} (the absolute page number),
+\texttt{counter} (the name of the current counter), and \texttt{xpos}
+and \texttt{ypos} (the position on the page as set by a previous
+\cs{pdfsavepos}).
   
-The module provides both \LaTeXe{} camel case commands and expl3 commands.    
+The module provides both \LaTeXe{} camel case commands and
+\LaTeX3 programming layer commands.    
 
 For documentation see \texttt{texdoc ltproperties-doc}. 
 
 \section{New or improved commands}
 
-\section{Testing for \pkg{expl3} version: \cs{IfExplAtLeastTF}}
-
-The integration of \pkg{expl3} into the kernel means that programmers can use
-all of the features available without needing to load explicitly. However, as
-\pkg{expl3} is not a separate package, the version can't be tested using
-\cs{IfPackageAtLeastTF}. To date, low-level methods have therefore been needed
-to check for the availablity of features in \pkg{expl3}. We have now added
-\cs{IfExplAtLeastTF} as a test working in the same way as
-\cs{IfPackageAtLeastTF} but focussed on the pre-loaded programming layer.
-Programmers can check the date of \pkg{expl3} they are using in the \texttt{.log},
-as it appears both at the start and end in the format
+\subsection{Testing for the \LaTeX3 programming layer version: \cs{IfExplAtLeastTF}}
+
+The integration of \pkg{expl3} (the \LaTeX3 programming layer)
+into the kernel means that programmers can use all of the features
+available without needing to load it explicitly. However, as \pkg{expl3}
+is not a separate package, the version can't be tested using
+\cs{IfPackageAtLeastTF}. To date, low-level methods have therefore
+been needed to check for the availability of features in
+\pkg{expl3}. We have now added \cs{IfExplAtLeastTF} as a test working
```

Comment:
I'd like to also mention that `expl3` is upgraded separately from LaTeX2e hence has its own version.

```suggestion
available without needing to load it explicitly. However, as \pkg{expl3}
is upgraded separately from \LaTeXe{} and it's not a separate package, its
version is different from that of \LaTeXe{} and can't be tested using
\cs{IfPackageAtLeastTF}. To date, low-level methods have therefore
been needed to check for the availability of features in
\pkg{expl3}. We have now added \cs{IfExplAtLeastTF} as a test working
```

Reply:
Done modulo some formality

Reply:
Just for the record, done in commit 84aeff78 (Improvement from Yukai Chou, 2023-10-26).